README.md

```diff
@@ -1,5 +1,32 @@
+# cli
+
+> Generated by [create-ink-app](https://github.com/vadimdemedes/create-ink-app)
+
 WORK IN PROGRESS
 
 CLI to work with your AsyncAPI files. You can validate them, use generator and even bootstrap a new file.
 
 Feel free to jump into the issues/PRs section and give your feedback on what we want to work on.
+
+## Install
+
+```bash
+$ npm install --global cli
+```
+
+
+## CLI
+
+```
+$ cli --help
+
+  Usage
+    $ cli
+
+  Options
+    --name  Your name
+
+  Examples
+    $ cli --name=Jane
+    Hello, Jane
+```
```

Comment:
```suggestion
```bash
$ asyncapi --help

  Usage
    $ asyncapi

  Options
    --name  Your name

  Examples
    $ asyncapi --name=Jane
    Hello, Jane
```
```

Reply:
Flags should be also update to the existing like `--context` and `--watch` :)